config/locales/es.yml

```diff
@@ -300,8 +300,11 @@ es:
       header: "dependencias de %{title}"
     gem_members:
       authors_header: Autores
+      not_using_mfa_warning_show:
```

Comment:
`not_using_mfa_warning_show: "* Algunos propietarios no están usando MFA. Haga click para ver la lista completa."`

`not_using_mfa_warning_show: "* Los siguientes propietarios no están usando MFA.  Haga click para ocultar."`

`using_mfa_info: "* Todos los propietarios están usando MFA."`

Reply:
Thanks! Added in 5cab56c